### PR TITLE
disable CI PR commenter

### PR DESCRIPTION
**Pull request by sateeshperi**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/methylseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/methylseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: workflows/methylseq.nf [hunk 1]
```diff
@@ -19,12 +20,21 @@ def checkPathParamList = [
...
+    } 
+}
 
+// Validate the mandatory parameter 'input'
+if (params.input) { 
+    ch_input = file(params.input) 
+} else { 
+    // If 'input' is not specified, terminate the program with an error message
+    exit 1, 'Input samplesheet not specified!' 
+}
```
Comment (Emiller88) [inline workflows/methylseq.nf:37]: Is this changing from the template? It's probably just easier to leave it as is.

**Review by Emiller88 — COMMENTED** [review]
Is this changing from the template? It's probably just easier to leave it as is.

Love the consistent whitespace changes, the comments though might make template sync more difficult than they're worth.

File: workflows/methylseq.nf (hunk 1 above)
Comment (sateeshperi) [reply workflows/methylseq.nf:37]: got ya

**Comment by sateeshperi** [comment]
i blame the co-pilot for that indulgence :joy:

**Comment by Emiller88** [comment]
Shh, don't let it hear you!

**Pull request merged**